SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -119,5 +120,19 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** The deployment to connect to when using the Development Authentication Flow. If left empty, it uses the first available one (order not guaranteed when there are multiple items). The deployment needs to be tagged with 'dev_login'. */
 	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (ConfigRestartRequired = false))
 	FString DevelopmentDeploymentToConnect;
-};
 
+	/** Single worker type launch when offloading is disabled, fallback worker type when offloading is enabled (owns all actor classes by default). */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading")
+	FWorkerType DefaultWorkerType;
+
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading")
+	bool bEnableOffloading;
+
+	/** Array of ActorGroups. */
```

Comment:
_comments a TMap saying it's an array_ :trollface: 

NIT could change to say something like "mapping from actor group name to worker type and actor classes in the group"

Reply:
Updated this and a few others. Since these are surfaced as tooltips in the editor feels redundant to mention it's a map since the UI makes that obvious?